package.json

```diff
@@ -249,7 +249,7 @@
     "test-precommit": "NODE_OPTIONS='--experimental-transform-types' node scripts/test.js --bail --findRelatedTests -u",
     "test-staged": "NODE_OPTIONS='--experimental-transform-types' node scripts/test.js --findRelatedTests $(git diff --name-only --cached)",
     "lint": "pnpm run lint:prettier && pnpm run lint:js && pnpm run lint:css",
-    "lint:js": "eslint",
```

Comment:
Does this specify concurrency for local dev and CI? If so, I wonder if it makes sense to split this up and increase it for local dev to maximize utilization.

Out of curiosity, how much slower is it if you set it to 4 (iirc gh runners have 4 cores, but from my exp, performance can vary between them by quite a lot)?

Reply:
it didn’t get any faster on my local machine with 4 as well. even `auto` wasn’t faster. I think we’re fine with 3 as a global setting for now.